skbio/__init__.py

```diff
@@ -4,6 +4,8 @@
 # Distributed under the terms of the Modified BSD License.
 #
 # The full license is in the file COPYING.txt, distributed with this software.
+#
+#scikit-bio's ASCII art tree was created by `@gregcaporaso`
```

Comment:
The build is failing because this isn't pep8 compliant (checklist.py will also complain b/c the copyright header needs to stay intact across all source files). Can you please move this comment directly above the `art` variable in this file?

There's also credits in the readme about the ASCII art title. Please move that directly above the `title` variable in this file.